src/javascript/FeverEffect.js

```diff
@@ -29,7 +37,8 @@ export default class FeverEffect {
     const cHeight = this.game.canvas.height;
     const num = cHeight / height + 2;
     const speed = fever.time - 10;
-    const alpha = fever.time / 80;
+    const feverAplahLevel = fever.time - 25 < 0 ? 0 : fever.time - 25;
+    const alpha = feverAplahLevel / 80;
     ctx.globalAlpha = alpha;
     for (let i = 0; i < num; i++) {
       ctx.fillStyle = "white";
```

Comment:
```suggestion
    ctx.fillStyle = "white";
    for (let i = 0; i < num; i++) {
```

Reply:
Unless I'm missing something, but is it necessary to execute `ctx.fillStyle = "white";` for `num` times in this `for` loop?

Reply:
Ahh yes.. here exists a weird bug where this fillStyle is overridden elsewhere, so I tried to put it inside the loop to stop that but it didn't work, I'll apply the change and find out why